packages/components/src/lib/controllers/app-controller.js

```diff
@@ -186,7 +186,6 @@ export default class AppController {
 
     // Subscribe to LexicalQuery events
     LexicalQuery.evt.LEXICAL_QUERY_COMPLETE.sub(appController.onLexicalQueryComplete.bind(appController))
-    LexicalQuery.evt.MORPH_DATA_READY.sub(appController.onMorphDataReady.bind(appController))
```

Comment:
We don't need this event any more. The only functionality that was related to it was a UI message. The message has been moved to the `HOMONYM_READY` handler , so the `MORP_DATA_READY` can be eliminated.